worq/core.py

```diff
@@ -138,7 +136,7 @@ def next_task(self, timeout=None):
             log.error('cannot deserialize task [%s:%s]',
                 self.name, task_id, exc_info=True)
             return None
-        #log.debug('next %s [%s:%s]', task.name, self.name, task_id)
+        # log.debug('next %s [%s:%s]', task.name, self.name, task_id)
```

Comment:
I don't believe this was a pep8 problem. My convention is to comment with no leading space for code that was commented out. Another option would be to remove this line entirely.